tools/rostest/src/rostest/rostest_parent.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, config, roslaunch_files, port=0):
             raise Exception("config not initialized")
         # we generate a run_id for each test
         run_id = roslaunch.core.generate_run_id()
-        super(ROSTestLaunchParent, self).__init__(run_id, roslaunch_files, is_core=False, is_rostest=True)
+        super(ROSTestLaunchParent, self).__init__(run_id, roslaunch_files, is_core=False, port=port, is_rostest=True)
```

Comment:
Please add new arguments at the end of the list in order to not break existing code.